core/math/geometry_3d.cpp

```diff
@@ -862,6 +862,7 @@ Vector<Vector3> Geometry3D::compute_convex_mesh_points(const Plane *p_planes, in
 }
 
 #define square(m_s) ((m_s) * (m_s))
+#define BIG_VAL 1e20
```

Comment:
Just to note these are both awful for global namespace.

If you must use such a define, it is better to `#undef` at the bottom of the file that uses it. But usually you can set as a const e.g. in `Geometry3D` if you want a local const.